routers/web/repo/pull.go

```diff
@@ -494,6 +494,7 @@ func PrepareMergedViewPullInfo(ctx *context.Context, issue *issues_model.Issue)
 		if len(commitStatuses) != 0 {
 			ctx.Data["LatestCommitStatuses"] = commitStatuses
 			ctx.Data["LatestCommitStatus"] = git_model.CalcCommitStatus(commitStatuses)
+			ctx.Data["ShowHideButton"] = true
```

Comment:
How `ShowHideButton` is used? I didn't see it in code.

Reply:
Looks unused, is it a leftover from previous iterations?

Reply:
Removed in https://github.com/go-gitea/gitea/pull/26284/commits/a91edb8f3fb59724fe028c2c89a4a47f2c95e035, no observable behaviour change.